db/migrate/20220727161425_create_lti_line_items.rb

```diff
@@ -0,0 +1,14 @@
+class CreateLtiLineItems < ActiveRecord::Migration[7.0]
+  def change
+    create_table :lti_line_items do |t|
+      t.string :lti_line_item_id, null: false
+      t.references :assessment, null: false, foreign_key: true
+      t.references :lti_deployment, null: false, foreign_key: true
+
+      t.timestamps
+    end
+    add_column :lti_deployments, :lms_course_id, :integer, null: true
```

Comment:
Why can the lms_course_id be null but the lms_course_name not be null?

Reply:
You're right, they should both be not nullable